PhotoMap/Features/Category/Coordinators/CategoryCoordinator.swift

```diff
@@ -14,8 +14,11 @@ class CategoryCoordinator: Coordinator {
     private let diContainer: DIContainerType
     private let cancelBag = CancelBag()
     
+    var parentCoordinator: (Coordinator & ReturnCategoriesWhenDonePressedProtocol)?
```

Comment:
Shouldn't it be a weak property?

Reply:
Also naming for `ReturnCategoriesWhenDonePressedProtocol` kind of not that good. Maybe just `CategoriesProtocol` or something?